SparseGrids/TasmanianSparseGridWrapC.cpp

```diff
@@ -108,6 +108,15 @@ void tsgMakeFourierGrid(void *grid, int dimensions, int outputs, int depth, cons
     if (depth_type == type_none){ depth_type = type_level; }
     ((TasmanianSparseGrid*) grid)->makeFourierGrid(dimensions, outputs, depth, depth_type, anisotropic_weights, limit_levels);
 }
+void tsgMakeGridFromCustomTabulated(void *grid, int dimension, int outputs, int depth, const char *sType, void *custom_tabulated,
+                                    const int *anisotropic_weights, const int *limit_levels) {
+    TypeDepth depth_type = IO::getDepthTypeString(sType);
+    #ifndef NDEBUG
+    if (depth_type == type_none){ cerr << "WARNING: incorrect depth type: " << sType << ", defaulting to type_iptotal." << endl; }
+    #endif // NDEBUG
+    TasGrid::CustomTabulated ct_copy = *reinterpret_cast<TasGrid::CustomTabulated*>(custom_tabulated);
```

Comment:
You can inline the copy and skip the extra variable:
```
((TasmanianSparseGrid*) grid)->makeGlobalGrid(dimension, outputs, depth, depth_type, CustomTabulated(ct_copy), anisotropic_weights, limit_levels);
```
The rest of the code is good.

Reply:
Done.